### PR TITLE
Course Acceptance Update Test and Course Controller actions complated

### PR DESCRIPTION
**Pull request by JaAfghan**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Acceptance/APIs/Courses/CoursesApiTests.Logic.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                await this.otripleSApiBroker.GetCourseByIdAsync(randomCourse.Id);
+
+            // then
+            actualCourse.Should().BeEquivalentTo(modifiedCourse);
+
+            await this.otripleSApiBroker.DeleteCourseByIdAsync(actualCourse.Id);
+        }
+
+        
+    }
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Acceptance/APIs/Courses/CoursesApiTests.Logic.cs:39]: Remove the extra new lines

File: OtripleS.Web.Api.Tests.Acceptance/APIs/Courses/CoursesApiTests.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
+﻿using OtripleS.Web.Api.Models.Courses;
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Acceptance/APIs/Courses/CoursesApiTests.cs:1]: Add copyrights notice here

**Comment by hassanhabib** [comment]
@tajwal this PR needs conflicts merge

**Comment by JaAfghan** [comment]
> @JaAfghan please address the conflicts in this PR

@hassanhabib  Conflict was in OtripleS.Web.Api/Controllers/CoursesController.cs 
in PostCourseAsync Method only persistedCourse variable was Conflict but in local i have already resolve it and i have copied Remote Changes in Local

**Comment by hassanhabib** [comment]
> > @JaAfghan please address the conflicts in this PR
> 
> @hassanhabib Conflict was in OtripleS.Web.Api/Controllers/CoursesController.cs
> in PostCourseAsync Method only persistedCourse variable was Conflict but in local i have already resolve it and i have copied Remote Changes in Local

@JaAfghan your branch still says conflicting files, I can't merge unless this is resolved.

**Comment by hassanhabib** [comment]
Shutting this PR down for being overdue - replaced with https://github.com/hassanhabib/OtripleS/pull/61

**Pull request closed without merging**